@xen-orchestra/fs/src/_encryptor.js

```diff
@@ -0,0 +1,136 @@
+const crypto = require('crypto')
+const { Transform } = require('stream')
+
+class Crypt extends Transform {
```

Comment:
Why not using directly [`Cypher`](https://nodejs.org/api/crypto.html#class-cipher)?

Reply:
I think it will not write the iv at the beginning of the stream, and I really want to keep this data with the file for decrypting.

Reply:
Maybe we can trick and use [`stream.unshift()`](https://nodejs.org/dist/latest/docs/api/stream.html#readableunshiftchunk-encoding).

I feel like the perf will be much better if we use it directly.